tla/consensus/MCccfraft.tla

```diff
@@ -143,12 +143,12 @@ DebugNotTooManySigsInv ==
 
 \* The inital log is up to 4 entries long + one log entry per request/reconfiguration + one signature per request/reconfiguration or new view (no consecutive sigs except across views)
 MaxLogLength == 
-    4 + ((RequestLimit + Len(Configurations)) * 2) + MaxTermLimit
+    4 + ((MaxRequestCount + Len(Configurations)) * 2) + (StartTerm + MaxTermCount)
```

Comment:
I think it should (famous last words) be ok to just do this (as the 4 already includes the starting entries
```suggestion
    4 + ((MaxRequestCount + Len(Configurations)) * 2) + MaxTermCount
```

Reply:
Agreed